governator-test-junit/src/main/java/com/netflix/governator/guice/test/AnnotationBasedTestInjectorManager.java

```diff
@@ -48,20 +51,19 @@
     private final SettableConfig classLevelOverrides = new DefaultSettableConfig();
```

Comment:
Do we really need to track these separately?  Why not just first populate class level overrides on a single SettableConfig and then apply the method overrides.